exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/ApiControllerIntegrationTest.java

```diff
@@ -340,6 +347,82 @@ void multiMapTest() {
     assertThat(m.get("k3")).isEqualTo("v3");
   }
 
+  @Test
+  void getHeight(VertxTestContext context) {
+    Height height = new Height(10L);
+
+    when(qaService.getHeight()).thenReturn(height);
+
+    get(BLOCKCHAIN_HEIGHT)
+        .send(context.succeeding(response -> context.verify(() -> {
+          assertThat(response.statusCode())
+              .isEqualTo(HTTP_OK);
+
+          String body = response.bodyAsString();
+          Height actualHeight = QaTransactionGson.instance()
+              .fromJson(body, Height.class);
+          assertThat(actualHeight).isEqualTo(height);
+
+          context.completeNow();
+        })));
+  }
+
+  @Test
+  void getHeight_throwsException(VertxTestContext context) {
+    when(qaService.getHeight()).thenThrow(new RuntimeException());
+
+    get(BLOCKCHAIN_HEIGHT)
+        .send(context.succeeding(response -> context.verify(() -> {
+          assertThat(response.statusCode())
+              .isEqualTo(HTTP_BAD_REQUEST);
```

Comment:
Do you think `bad request` should be returned in case of an exception? It seems to me that there is nothing wrong with the request 🤔 Maybe `422` would be better or even something else?
Although it is not connected to your task so it's not necessary to look into that right now :)

Reply:
Good question. Let's think...
> @throws RuntimeException if the "genesis block" was not created
So in case, you get this error it means you're doing something wrong i.e. sending the request too early. But mb not catching it will be a better solution? 🤔

Reply:
Both `bad request` status and the fact that user is expected to catch an unchecked exception seem strange to me.

Reply:
@MakarovS we can get rid of `bad request` and return `500`.  What do you think?
But `RuntimeException` is ok for me. It is a really rare case with genesis block.

Reply:
Ok, let's leave it as it is for now and we'll get back to it later.